Extension/src/LanguageServer/client.ts

```diff
@@ -3502,12 +3505,27 @@ export class DefaultClient implements Client {
                     character: range.end.character,
                     line: range.end.line
                 }
-            }
+            },
+            copyToClipboard: false
         };
 
+        if (copy) {
+            params.copyToClipboard = true;
+        }
+
         const result: CreateDeclarationOrDefinitionResult = await this.languageClient.sendRequest(CreateDeclarationOrDefinitionRequest, params);
         // TODO: return specific errors info in result.
-        if (result.changes === undefined) {
+        if (result.edit.changes.length === undefined && result.clipboardText) {
+            if (!params.copyToClipboard) {
+                util.promptCDDFailed();
+            } else {
+                vscode.env.clipboard.writeText(result.clipboardText).then(() => {
```

Comment:
Can this just be `vscode.env.clipboard.writeText(result.clipboardText);`? It seems unnecessary to handle a failure, since that is not expected and would be some VS Code issue.

Reply:
I think it is necessary to notify the user in this case since this is not a general copy to clipboard scenario but rather it is invoked via our extension.

Reply:
I don't know what the difference is -- the vscode.env.clipboard.writeText is an API that our extension invokes and we don't check for failure in our previous usage of it. How/why would copy to clipboard fail?

Reply:
@bobbrow @Colengms @michelleangela @AlexandraKemperMS what do we think about this?

Reply:
I would avoid using `.then` in a method already marked `async`.  You should use:
```ts
  try {
    await vscode.env.clipoard.writeText(result.clipboardText);
  } catch {
    // handle failure
  }

Reply:
But do you support the concept of catching the error in the first place? @bobbrow

Reply:
I'm not opposed. If it never fails no one will ever see the response to the failure.

Reply:
It's difficult to tell from the documentation because it doesn't specify, but the contract for `Thenable` is that it is allowed to reject/throw.  It's probably unlikely that it will fail, but we don't have any global try/catch wrappers to see which vscode API's are throwing that we may not be handling.